From 21bc211143a5f595e33b470d9997e1ab0f37656b Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Sat, 20 Jan 2024 10:44:02 -0800
Subject: [PATCH 1/2] disable precompiled hip headers

- precompiled headers can cause
  problems on gfx10 cards
- this code has usage also in openmi,
  so verify that nothing in there does
  not try to use the disabled features.

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 hipamd/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hipamd/CMakeLists.txt b/hipamd/CMakeLists.txt
index ccfbceef6..8db396386 100755
--- a/hipamd/CMakeLists.txt
+++ b/hipamd/CMakeLists.txt
@@ -43,7 +43,7 @@ list(APPEND CMAKE_MODULE_PATH ${HIP_COMMON_DIR}/cmake)
 #############################
 # Options
 #############################
-option(__HIP_ENABLE_PCH "Enable/Disable pre-compiled hip headers" ON)
+option(__HIP_ENABLE_PCH "Enable/Disable pre-compiled hip headers" OFF)
 option(HIP_OFFICIAL_BUILD "Enable/Disable for mainline/staging builds" ON)
 option(FILE_REORG_BACKWARD_COMPATIBILITY "Enable File Reorg with backward compatibility" OFF)
 
-- 
2.41.0

